libcontainer/factory_nabla.go

```diff
@@ -86,6 +87,18 @@ func applyPauseHack(config *configs.Config, containerRoot string) (*configs.Conf
 func createRootfsISO(config *configs.Config, containerRoot string) (string, error) {
 	rootfsPath := config.Rootfs
 	targetISOPath := filepath.Join(containerRoot, "rootfs.iso")
+	for _, mount := range config.Mounts {
+		if (mount.Destination == "/etc/resolv.conf") ||
+			(mount.Destination == "/etc/hosts") ||
+			(mount.Destination == "/etc/hostname") {
+			var dest = filepath.Join(rootfsPath, mount.Destination)
+			var source = mount.Source
+			var err = utils.Copy(dest, source)
+			if err != nil {
+				return "", errors.Wrap(err, "Unable to copy " + source + " to " + dest)
+			}
```

Comment:
golang Style nit: 

- Use `dest :=` instantiation instead of `var dest =`
- `if err := utils.Copy(dest, source); err != nil { ... }`